MovieDatabase/MovieDatabase/Controllers/MovieController.cs

```diff
@@ -0,0 +1,27 @@
+using Microsoft.AspNetCore.Mvc;
+using MovieDatabase.Interfaces;
+using MovieDatabase.Models;
+
+namespace MovieDatabase.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class MovieController : Controller
+    {
+        private readonly IMovieRepository _MovieRepository;
+        public MovieController(IMovieRepository MovieRepository)
+        {
+            _MovieRepository = MovieRepository;
+        }
+        [HttpGet]
+        [ProducesResponseType(200, Type=typeof(IEnumerable<Movie>))]
+        public IActionResult GetMovies()
```

Comment:
can you also add get by id, in the repository you will have a new method similar to the one that you have now but you will make a where check to check for the given ID which will be an input to the method and instead of returning ToList(), we return with FirstOrDefault()

Reply:
add this for all other endpoints too